src/prompts/triggers.spec.ts

```diff
@@ -23,10 +28,10 @@ test.each(cases)(
     }
 );
 
-function newPageViewPrompt(pathFilter: string): PromptConfig {
+function newPageViewPrompt(pathFilter: string): Partial<PromptConfig> {
```

Comment:
Why is it necessary to declare this as a partial now?

Reply:
Due to non-optional label/colours config keys in base prompt.

Solved by removing these from base as they were explicitly declared for every subtype anyway.